uber/templates/base.html

```diff
@@ -165,7 +165,10 @@
     {% block head_additional %}{% endblock %}
 </head>
 <body>
-    {% if c.PRE_CON %}<div class="loader"><a class="loader_link" href="../static_views/slow_load.html" target="_blank"></a></div>{% endif %}
+    {% if c.PRE_CON %}
+        <div class="loader"><a class="loader_link" href="../static_views/slow_load.html" target="_blank"></a></div>
+        <script src="https://maps.googleapis.com/maps/api/js?sensor=false&amp;libraries=places"></script>
```

Comment:
The ``&amp;`` is almost certainly not what you want here, and may adversely affect what gets loaded.